openlibrary/macros/SearchResultsWork.html

```diff
@@ -91,26 +91,28 @@ <h3 itemprop="name" class="booktitle">
             $:macros.BookByline(author_data, limit=max_rendered_authors, overflow_url=work_edition_url, attrs='class="results"')
         </span>
         <span class="resultPublisher">
+          $if doc.get('ratings_count') and doc.get('ratings_average') and doc.get('want_to_read_count'):
+            $:macros.StarRatingsByline(doc)
           $if doc.get('first_publish_year'):
             <span class="publishedYear">
               $_('First published in %(year)s', year=doc.first_publish_year)
             </span>
           $if doc.get('edition_count'):
             <a href="$work_edition_all_url#editions-list">$ungettext('%(count)s edition', '%(count)s editions', doc.edition_count, count=doc.edition_count)</a>
-            $if doc.get('languages'):
-              <span class="languages">
-                $:ungettext('in <a class="hoverlink" title="%(langs)s">%(count)d language</a>', 'in <a class="hoverlink" title="%(langs)s">%(count)d languages</a>', len(doc.languages), count=len(doc.languages), langs=commify_list([get_language_name('/languages/' + lang) for lang in doc.languages]))
+          $if doc.get('languages'):
+            <span class="languages">
+              $:ungettext('in <a class="hoverlink" title="%(langs)s">%(count)d language</a>', 'in <a class="hoverlink" title="%(langs)s">%(count)d languages</a>', len(doc.languages), count=len(doc.languages), langs=commify_list([get_language_name('/languages/' + lang) for lang in doc.languages]))
```

Comment:
Full Text Searching for the fun explosions in the sky album name "all of a sudden i miss everyone" results in a bug

https://testing.openlibrary.org/search/inside?q=all+of+a+sudden+i+miss+everyone&debug=true

<img width="1254" alt="Screenshot 2024-08-26 at 10 33 50 AM" src="https://github.com/user-attachments/assets/a97336f9-2e76-4170-b21c-7a60cb5804fd">